### PR TITLE
chore (DDI-1244): Update app header examples to include micro site header

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: apps/angular-demo/src/app/dropdown/dropdown.component.html [hunk 1]
```diff
@@ -109,8 +109,10 @@ <h1>Margin Spacing</h1>
 
 <h2>Reactive Binding</h2>
 <div>Selected Color: {{ reactiveFormCtrl.value }}</div>
-<goa-button (_click)="reactiveFormCtrl.setValue('red')" type="tertiary">Update to red</goa-button>
-<br/>
+<goa-button (_click)="reactiveFormCtrl.setValue('red')" type="tertiary"
```
Comment (chrisolsen) [inline apps/angular-demo/src/app/dropdown/dropdown.component.html:112]: Are these formatting changes the result of the pre-push format or of some auto-formatting within your editor?

Comment (ArakTaiRoth) [reply apps/angular-demo/src/app/dropdown/dropdown.component.html:112]: @chrisolsen This has been answered. If you're good, can you please resolve this and approve?

**Pull request merged**